### ISSUE TITLE
Fix scrolling

### ISSUE THREAD

**Issue by rafelafrance** [MapofLife/datasets#38]
Have the facets and map/list areas scroll independently

**Comment by jmalczyk** [comment]
This is better. I like that the facets stay in view. What do you think
@funkycoda? Was originally thinking of using display:fixed on the sidebar
content to keep it in view as the rest of the page scrolled, so the header
could scroll out of view, but this is probably fine.

**Comment by funkycoda** [comment]
I agree. I do like this already, but  future version should scroll the header. There isn't too much content within header to let it NOT scroll. This would mean it would have the same functionality as the dataset info page too where the header scrolls.

E.g: http://mol.org/datasets/fe4b9fd2-7da6-46b8-8bef-754044461210

**Comment by jmalczyk** [comment]
yeah, maybe while we are here, see  if you can get it so everything scrolls
as before, but the sidebar container   sticks to the top when the header is
out of view. There may be a bootstrap class for this.

**Comment by funkycoda** [comment]
Yeah, it'd be great to see if we can get this affect: http://getbootstrap.com/javascript/#affix 
Can't see it available on [Angular-Bootstrap](http://angular-ui.github.io/bootstrap/)

Don't think this is super urgent though. Something for a wish list, perhaps.

**Comment by jmalczyk** [comment]
If it involves significant jquery-based bootstrap let's not touch it right
now.
